src/shaders/glsl/ShaderOpMult.hpp.in

```diff
@@ -0,0 +1,101 @@
+#pragma once
```

Comment:
Why is the file called `.hpp.in`? instead of just `.hpp`?

Reply:
Ah, because it will be copied as a replacement in case compiling headers is disabled. To make it clear, this is not a valid file a user should compile with, I appended the `.in`.

Reply:
Ah ok I see, thanks for clarifying, this makes sense